lib/benchee/formatters/unit.ex

```diff
@@ -28,21 +35,27 @@ defmodule Benchee.Unit do
 
   @callback magnitude(unit) :: number
 
-  def float_precision(float) when float < 0.01, do: 5
-  def float_precision(float) when float < 0.1, do: 4
-  def float_precision(float) when float < 0.2, do: 3
-  def float_precision(_float), do: 2
+  @doc """
+  A separator that should appear between a value and a unit label. For example,
+  a space: `5.67 M` or an empty string: `5.67M`
+  """
+  @callback separator :: String.t
```

Comment:
btw. I'm open to changing it back to be directly attached to the number... for some reason I thought a space was more common.. but they also do it inconsistently e.g. i/s has a space while M/K don't. I also feel like `M/K` don't need one but `Million` would need to have one. Weird, huh? :)